recipes/perl-class-accessor/recipe.yaml

```
@@ -0,0 +1,48 @@
+schema_version: 1
+
+context:
+  name: perl-class-accessor
+  version: 0.51
+  sha256: bf12a3e5de5a2c6e8a447b364f4f5a050bf74624c56e315022ae7992ff2f411c
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/K/KA/KASEI/Class-Accessor-${{ version }}.tar.gz
+  sha256: ${{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - perl Makefile.PL INSTALLDIRS=site NO_PERLLOCAL=1 NO_PACKLIST=1
+    - make
+    - make test
+    - make install
+
+requirements:
+  build:
+    - make
+  host:
+    - perl=5.32
+  run:
+    - perl>=5.32
+
+tests:
+  - perl:
+      uses:
+        - Class::Accessor
+        - Class::Accessor::Fast
+        - Class::Accessor::Faster
+
+about:
+  license: perl_5
+  summary: Automated accessor generation
+  homepage: http://metacpan.org/pod/Class-Accessor
+
+extra:
+  recipe-maintainers:
+    - danielnachun
+    - anjos
```

```suggestion
    - anjos
    - conda-forge/perl-packagers
```
Let's also add `perl-packagers` as a maintainer for all these modules so that in the future others can get maintenance access to all the modules by joining that organization.